modules/games/definitions/neutreeko/instructions.ts

```diff
@@ -0,0 +1,18 @@
+import { NeutreekoDefinition } from "./_types";
+
+const neutreekoInstructions: NeutreekoDefinition["instructions"] = {
+  startTurn: { line: ["Select", { unittype: ["pieces", ["player"]]}, "to move"]},
```

Comment:
I found a shortcut for this that I had forgotten about!

```suggestion
  startTurn: { line: ["Select", "pieces", "to move"]},
```

A string matching a unit group will be replaces with `{ unittype: ["<group>", ["player"]]}`.

I could use this shortcut in a whole bunch of places, I had completely forgotten about it. :P

Reply:
Ah cool!